packages/web/src/analytics/analytics.ts

```diff
@@ -2,41 +2,31 @@ import Cookies from 'js-cookie'
 import getConfig from 'next/config'
 import { isBrowser } from 'src/utils/utils'
 
-let analytics: {
-  track: (key: string, properties?: object, options?: object) => void
-  page: () => void
-}
+const { publicRuntimeConfig } = getConfig()
```

Comment:
hmm whenever ive tried to call this top level (not in a function) its ended up crashing. 

Maybe the recent upgrade fixed that. 🤔

Reply:
Hm, I haven't had any issues with it... Is it crashing currently? Or it just has in the past?

Reply:
just run it for myself seem to be working.  know this now works is a great find.